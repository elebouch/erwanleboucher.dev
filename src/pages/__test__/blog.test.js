import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Blog from "../blog"

describe("Blog", () => {
  it("renders correctly", () => {
    const { container } = render(<Blog />)
    expect(container.innerHTML).toMatchSnapshot()
  })
})
