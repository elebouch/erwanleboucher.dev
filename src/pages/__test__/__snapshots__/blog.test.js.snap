// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog renders correctly 1`] = `"<article><div class=\\"sc-AxhUy hSpNqt\\"><div class=\\"sc-AxhCb cXBSVz\\"></div><span class=\\"sc-AxgMl eEdZGe\\">Blog Posts</span><h2>Coming soon ...</h2></div><div class=\\"sc-AxhUy hSpNqt\\"><div class=\\"sc-AxhCb cXBSVz\\"></div><span class=\\"sc-AxgMl eEdZGe\\">External Posts</span><a href=\\"https://bit.ly/2YDhxDJ\\"><div class=\\"sc-AxheI jmVYRX\\"><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">Comment fonctionne une Rom Custom?</p><p class=\\"sc-AxiKw icjPNi\\">Oct 6, 2017</p><p>Vous devez sûrement entendre, si vous intéressez à Android, les mots “Rom Custom”, “CyanogenMod”, “LineageOS”… Mais vous avez aucune idée de ce qu’il s’agit. Voilà un article qui pourra, vous éclairez !</p></div></div></a><a href=\\"https://bit.ly/3eFpEFw\\"><div class=\\"sc-AxheI jmVYRX\\"><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">Kernel et Android : Qu’est-ce que c’est et pourquoi le modifier ?</p><p class=\\"sc-AxiKw icjPNi\\">Feb 16, 2017</p><p>Vous êtes très nombreux à bidouiller votre appareil Android. Nous nous sommes intéressés à un composant essentiel de votre système : le kernel.</p></div><img src=\\"https://images.frandroid.com/wp-content/uploads/2017/02/android-kernel-explication.jpg\\" loading=\\"lazy\\" height=\\"100\\" style=\\"border-radius: 15px;\\"></div></a></div></article>"`;
